client/src/fabric/IFabricWallet.ts

```diff
@@ -17,6 +17,8 @@ export interface IFabricWallet {
 
     importIdentity(certificate: string, privateKey: string, identityName: string, mspid: string): Promise<void>;
 
+    delete(identityName: string): Promise<void>;
```

Comment:
where's the function associated with this?

Reply:
@heatherlp same place as exists - from the base FileSystemWallet class.